scripts/docker/web/elektrad-demo/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# same as `elektrad`, but with a KDB config set up (for demo)
+
+FROM elektra/web-base:latest
+
+WORKDIR /home/elektra
+
+# prepare demo environment
+
+# mount copy of /etc/hosts to user/hosts
+
+# mount as root user
+RUN kdb mount --with-recommends hosts user/hosts hosts
```

Comment:
will that work? shouldn't you be the elektra user when inheriting from your base image?

Reply:
yes, `USER` is not called in the base image